partner_contact_depth/data/system_parameter.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="contacts_max_depth" model="ir.config_parameter">
```

Comment:
IHMO, it should not be possible to update the configuration parameter if there is any partner which will then trigger the ValidationError.
WDYT?